curve-ansible/roles/install_package/tasks/include/install_nebd.yml

```diff
@@ -64,6 +64,12 @@
         local_file_path: "{{ local_nebd_package_path }}/bin/"
         file_mode: 0755
       include_tasks: copy_file_to_remote.yml
+    - name: install nebd include
+      vars:
+        remote_dir_name: "{{ curve_include_dir }}"
+        local_file_path: "{{ local_nebd_package_path }}/include/"
```

Comment:
The 'include' directory doesn't exist in nebd-package.

Reply:
it's been created in mk-tar.sh

https://github.com/opencurve/curve/blob/32ee88c706dacca40d2118bad41310de0e92cf93/mk-tar.sh#L457